tests/components/mqtt/test_cover.py

```diff
@@ -387,40 +402,37 @@
     state_attributes_dict = hass.states.get("cover.test").attributes
     assert not ("current_position" in state_attributes_dict)
     assert not ("current_tilt_position" in state_attributes_dict)
-    assert not (4 & hass.states.get("cover.test").attributes["supported_features"] == 4)
+    assert not (
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4636028)